src/platform/web/ThemeLoader.ts

```diff
@@ -75,7 +75,12 @@ export class ThemeLoader {
             const builtAssets: Record<string, string> = manifest.source?.["built-assets"];
             const themeName = manifest.name;
             let defaultDarkVariant: any = {}, defaultLightVariant: any = {};
-            for (const [themeId, cssLocation] of Object.entries(builtAssets)) {
+            for (let [themeId, cssLocation] of Object.entries(builtAssets)) {
+                /**
+                 * This cssLocation is relative to the location of the manifest file.
+                 * So we first need to resolve it relative to the root of this hydrogen instance.
+                 */
+                cssLocation = new URL(cssLocation, new URL(manifestLocation, window.location.origin)).href;
```

Comment:
I think this can throw, we should probably still try any next items in the builtAssets when that happens?

Reply:
Fixed by https://github.com/vector-im/hydrogen-web/pull/760/commits/6718198d9cc9e5aac8d9e3c0ca5534440b53bdcd